omod/src/main/web/dashboardwidgets/obsacrossencounters/obsacrossencounters.spec.js

```diff
@@ -264,6 +264,48 @@ const mockEncounters = [
                 "groupMembers": null
             }
         ]
+    },
+    {
```

Comment:
Please update the comment at the top of this block. I'd add

```
Report
- uuid-limb-text
- uuid-limb-text
```

Reply:
Thanks, I'll add that!